proto/wippersnapper/display/v1/display.proto

```diff
@@ -0,0 +1,144 @@
+// SPDX-FileCopyrightText: 2025 Brent Rubell for Adafruit Industries
+// SPDX-License-Identifier: MIT
+// Display API for Adafruit WipperSnapper
+
+syntax = "proto3";
+
+package wippersnapper.display.v1;
+import "nanopb/nanopb.proto";
+import "wippersnapper/i2c/v1/i2c.proto";
```

Comment:
is this import still needed? (protoc is throwing a warning that it is unused)

Reply:
@lorennorman  Unused, you can remove it.